modules/salt/pages/formulas-suma.adoc

```diff
@@ -0,0 +1,56 @@
+[[formulas-suma]]
+= Formulas Provided by {productname}
+
+Some formulas are provided by default with {productname}.
+Other official formulas can be installed as RPM packages.
+When the formula is installed, you can activate them using the {productname} {webui}.
+
+For information about writing custom formulas, see xref:salt:formulas-custom.adoc[].
+
+
+== Install Formulas from an RPM
+
+{suse} releases formulas as RPM packages.
+Formulas are provided in the {productname} pool software channel.
+
+
+[IMPORTANT]
+====
+If a formula uses the same name as an existing Salt state, the two names will collide, and could result in the formula being used instead of the state.
+Always check states and formulas to avoid name clashes.
+====
+
+
+
+.Procedure: Installing Formulas from an RPM
+. On the {productname} Server, at the command prompt, search for available formulas:
++
+----
+zypper se --type package formula
+----
+. Get more information about a formula:
++
+----
+zypper info locale-formula
+----
+. On the {productname} Server, at the command prompt, as root, install the formula:
++
+----
+zypper in locale-formula
+----
+
+
+
+== Activate Formulas from the {webui}
+
+Formulas provided by {productname}, or formulas that you have installed, can be activated using the {productname} {webui}.
+
+
+.Procedure: Activate Formulas from the {webui}
+. In the {productname} {webui}, navigate to menu:Systems[List], select the client you want to activate the formula for.
+. Navigate to the menu:Systems[Formulas] tab, and check the formula you want to activate.
+. Click btn:[Save].
+. Navigate to the new subtab for the formula, and configure the formula as required.
+. Apply the highstate.
+
+More specific information about each of the {productname} formulas is available in this guide.
```

Comment:
where? :wink:

Reply:
They're all in individual files, and this is the top level page. So normally I would link to the top level page, but in this case it would be self-referential. Not sure what else to do really :woman_shrugging:

Reply:
Ok, I see the dilemma.  If this is the top level page, we probably do not need to mention this matter of fact at all, of we could say, "see xref:formula_x and the following sections".  Or just list them all here (minitoc) ;-P . Maybe, sth to think about later.

Reply:
I feel like either of those options would carry a maintenance burden. I'll just take the sentence out. Hopefully it's obvious that there are subsections :wink: